README.adoc

```diff
@@ -1,91 +1,99 @@
 = Spring Cloud Data Flow Acceptance Tests =
 
-This project bootstraps a dataflow server on a target platform, executes a series of tests by creating a series of streams and tasks and then cleans up after its done.
+This project bootstraps a dataflow server on a target platform (certified local, kubernetes, cloudfoundry environments), executes a series of tests by creating a series of streams and tasks and then cleans up after its done.
```

Comment:
nit pick - but maybe drop "certified"

Reply:
Maybe supported environments is better. I mean, we don't intend to test on every release of every platform.

Reply:
sure that's fine